pkg/services/ngalert/store/alertmanager.go

```diff
@@ -82,6 +82,10 @@ func (st DBstore) SaveAlertmanagerConfigurationWithCallback(ctx context.Context,
 		if _, err := sess.Insert(config); err != nil {
 			return err
 		}
+
+		// Add the created ID to the command.
+		cmd.ResultID = config.ID
+
```

Comment:
IMO this line alone deserves a separate test in `alertmanager_test` (or a new assert in an existing test) since it's a change in the API of a _really_ commonly used function. Without a test, this is the kind of thing that is easily broken since it's not an obvious behavior change and not enforced by the compiler.